packages/nc-gui/components/project/table.vue

```diff
@@ -109,12 +109,14 @@ export default {
           const relationColumns = meta.columns.filter(c => c.uidt === UITypes.LinkToAnotherRecord)
 
           if (relationColumns.length) {
-            const referredTables = await Promise.all(relationColumns.map(async(c) => {
+            const refColMsgs = await Promise.all(relationColumns.map(async(c, i) => {
               const refMeta = await this.$store.dispatch('meta/ActLoadMeta', { id: c.colOptions.fk_related_model_id })
-              return (refMeta && refMeta.title) || c.title
+              return `${i + 1}. ${c.title} is a LinkToAnotherRecord of a ${(refMeta && refMeta.title) || c.title}`
```

Comment:
I think `a` in `of a ${(refMeta && refMeta.title) || c.title}` is unncessary.

![image](https://user-images.githubusercontent.com/35857179/165671403-0cba0a28-cdf8-43d4-8b11-c55b81324ea1.png)